src/evox/algorithms/mo/nsga3.py

```diff
@@ -0,0 +1,173 @@
+import evox as ex
```

Comment:
currently, it's recommanded to just `import evox`, since evox is already short enough.
or, u can do `from evox import jit_class, Algorithm, State`